packages/fields/src/types/CloudinaryImage/views/blocks/single-image.js

```diff
@@ -0,0 +1,213 @@
+/** @jsx jsx */
```

Comment:
This file is mostly a copy+paste of the `image-container` block from the `Content` field with exceptions noted below.